src/index.html

```diff
@@ -7,8 +7,54 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#logo" class="nav_link_logo">
+        <img src="./images/logo.png" alt="logo"
+        class="nav_img_logo">
+      </a>
+
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a href="#apple" class="nav__link is-active" >apple</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#samsung" class="nav__link">samsung</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#smartphones" class="nav__link">smartphones</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#laptops&computers" 
+            class="nav__link"
+            data-qa="hover" >laptops & computers</a>
```

Comment:
Pay attention on code style in case when tag has 3 and more attributes
```suggestion
            <a 
              href="#laptops&computers" 
              class="nav__link"
              data-qa="hover"
            >
              laptops & computers
            </a>
```